### DISCUSSION TITLE
strang readout

### DISCUSSION THREAD

**Discussion by squid-slime** [MilkJug1/OculusLinkLinux#16] [General]
in linux terminal "cd" to oculussetup.exe, type "wine cmd" now should display as C:\. type "OculusSetUp.exe /DisableCertCheck" 

my result https://pastebin.mozilla.org/u1RcZtyS

other commands but use "/" and not "--"

```

--unattended

  --diagnostic

  --uninstall

  --repair

  --noupdates

  --bypassHotfixCheck

  --deleteUserData

  --locale

  --drive

  --sandbox

  --setupPath

  --DisableCertRevocationCheck

  --DisableCertCheck

  --RepairRuntimeOnly

  --SourceFolder

  --help                          Display this help screen.

  --version                       Display version information.
```

**Comment by MilkJug1** [comment]
Wait, this is all of the parameters that the OculusSetup.exe file has? Weird thing to have it forcefully use the "/" character instead of the "-" character(even though the help screen says to use "-")

**Reply by MilkJug1** [reply]
I will try to run this through wine cmd, whenever I can since wine cmd is hanging everytime i run the command in the console(just the wine cmd, havent been able to type out the OculusSetup.exe thing yet).

**Comment by MilkJug1** [comment]
Ok, decided to do this and it came back with some intresting results.
Number 1, I decided to use the params
`OculusSetup.exe --DisableCertCheck --DisableCertRevocationCheck --drive Z:`
due to the fact that, I need Oculus to not check for anything and just try to install, but the `--drive Z;` param fixed the "Not Enough Space" issue but created another issue with the screen of "Ineligible Drive Selected".
![image](https://user-images.githubusercontent.com/76970674/170834609-8b1463fd-16fa-4b1a-9480-471da4b8179f.png)

Looking through some of the output that wine gave me of why its not installing, my first thought is that it could be that it doesn't think of my drive as the C: drive, it thinks of it as it the Z: drive for some reason. Ill post a gist here sometime soon for the entire thing.

**Reply by MilkJug1** [reply]
[This is the debug output that I keep getting when running this.](https://gist.github.com/MilkJug1/f75be400f0fcf7818393c6be998dac2f)

Its the gist that I was talking about, Wine is making all of the drives like 4 GB in size as denoted by this image here: 
![image](https://user-images.githubusercontent.com/76970674/170835183-06685706-fae1-4726-9e36-b69f90c37af0.png)
I don't know why wine keeps doing this, if we are able to change the size of the containers than maybe this could correctly install.
If we can't than I would have to go off of a already existing install of Oculus.
